tests/test-data/files/small-files/replace-data/remove_filter_before.meta.csv

```diff
@@ -0,0 +1,6 @@
+col_name,col_type,label,indicator_grouping,indicator_unit,indicator_dp,filter_hint,filter_grouping_column
+pupil_sen_status,Filter,SEN provision,,,,EHC plan/Statement of SEN or SEN support/School Action/School Action plus,
+primary_need,Filter,Primary type of need,,,,"(ASD, HI, VI�)",
```

Comment:
Is there a problem with the encoding of these files? Do they need to be saved as utf-8?
Same comment in `tests/test-data/files/small-files/replace-data/remove_filter_item_after_visual_impair.meta.csv`

Reply:
Strictly, I think you're right. I created these files by copying and editing existing test data sets we have, so they'll also have this issue.

But practically, I don't think it's worth the effort to fix this - the files import and work and serve their purpose for testing. I don't think we actually gain anything by fixing this. If it becomes an issue in the future, we could fix it then.

Reply:
Cool, will approve it. It would have been nice not to have this problem in the data though which proliferates with copying and pasting 🙂 .